pkgs/tools/backup/znapzend/default.nix

```
@@ -21,6 +21,11 @@ let
     propagatedBuildInputs = [ perlPackages.IOPipely Mojolicious-8-35 ];
   };
 
+  perl' = perl.withPackages (p:
+    [ Mojolicious-8-35 MojoIOLoopForkCall-0-20
+      p.TAPParserSourceHandlerpgTAP p.IOPipely
```

```suggestion
    [ MojoIOLoopForkCall-0-20 p.TAPParserSourceHandlerpgTAP
```


`p.IOPipely` and `Mojolicious-8-35` are already here via `MojoIOLoopForkCall-0-20`

`perl.withPackages` magically brings all the deps which one had to explicitly put into `$PERL5LIB`.
There could be more places like that (probably all of the `$PERL5LIB`->`perl.withPackages` replacements)

---

Oh right, thank you for the remainder. Fixed it.